hazelcast-integration/springboot-caching/.gitignore

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
@mustafaiman It's a matter of taste. Just want to have everything I need in place. I prefer don't mess with global things in a local commit. Do you think it does make sense to move those ignores to global (project wise) config (in separate PR)?